Error404 Jon

jpecinagithub opened the pull request.

jpecinagithub committed:
Footer Jon

Archivos html y css

jpecinagithub committed:
pagina no encontrada

gmarcg00 commented:
Lo que es la estructura y diseño html lo veo bien, aunque hay un par de cosas a tener en cuenta. Cambiaría el nombre de la página a not-found, al igual que la ruta.

Por otro lado, hay que tener cuidado a la hora de trabajar con diferentes ramas, si te fijas, en esta pull request están apareciendo los ficheros que añadiste para crear el footer (los iconos, el html, css). Esto ha pasado porque primero empezaste con el footer e hiciste el proceso de forma correcta. Pero cuando luego quisiste empezar con la página de not-found (error404) creaste la rama a partir de la del footer y desde develop. Esto provoca que tengas lo que hiciste en el footer en las dos prs.

El proceso correcto habría sido una vez creada la pull request del footer, cambiarse a la rama develop y crear otra nueva rama desde ahí.

Para solucionarlo, tienes que situarte en la rama error404Jon, borrar todos los ficheros del footer y subir los cambios.

jpecinagithub committed:
Modificació nombre archivo

Hemos camciado el nombre del archivo de error404 a not-found. Hemos actualizado el fichero de routes.

jpecinagithub commented:
> lucionarlo, tienes que situarte en la rama error404Jon, borrar todos los ficheros del footer y subir los cambi

Gracias Guille, no tengo muy claro de lo borrar archivos de una rama. Creo que entiendo cual ha sido el problema, pero no tengo claro como solucionarlo.

gmarcg00 commented:
> > lucionarlo, tienes que situarte en la rama error404Jon, borrar todos los ficheros del footer y subir los cambi
> 
> Gracias Guille, no tengo muy claro de lo borrar archivos de una rama. Creo que entiendo cual ha sido el problema, pero no tengo claro como solucionarlo.

gmarcg00 commented:
> > lucionarlo, tienes que situarte en la rama error404Jon, borrar todos los ficheros del footer y subir los cambi
> 
> Gracias Guille, no tengo muy claro de lo borrar archivos de una rama. Creo que entiendo cual ha sido el problema, pero no tengo claro como solucionarlo.

De la misma forma que creas nuevos ficheros, cuando vas a git kraken te aparecen esos nuevos ficheros para que los muevas a la columna de stagged. Cuando tu eliminas un fichero en tu local, en git kraken tambiién te aparece ese fichero para que lo muevas a la columna de stagged. La diferencia es que en el primer caso lo que has hecho es registrar el cambio de la creación del archivo, y en el segundo caso lo que quieres es registrar el cambio de la eliminación del fichero

jpecinagithub committed:
Modificacion pagina not found

jpecinagithub committed:
Modificación pagina not found
